### ISSUE TITLE
version.py 

### ISSUE THREAD

**Issue by ndombrowski** [ythuang0522/homopolish#64]
I have installed homopolish via the environmental.yaml as the conda install failed. This works perfectly, but when using homopolish --version I get 0.3.4 and not 0.4.1. 

I assume that is because the version in https://github.com/ythuang0522/homopolish/blob/master/version.py was not changed to reflect to 0.4.1 I but wanted to confirm if that is correct?

**Comment by StephDC** [comment]
That is correct. Apparently not only in the master branch, the 0.4.1 tag also had the version=0.3.4 left unchanged.

https://github.com/ythuang0522/homopolish/blob/7d47b289a56cd0c65f96c5b29f48890a2363f636/version.py#L1
